integration/kubernetes/e2e_conformance/run.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This script runs the Sonobuoy e2e Conformance tests.
+# Run this script once your K8s cluster is running.
+# WARNING: it is prefered to use containerd as the 
+# runtime interface instead of cri-o as we have seen
+# errors with cri-o that still need to be debugged.
+
+set -e
+
+export KUBECONFIG=$HOME/.kube/config
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/../../../lib/common.bash"
+
+RUNTIME="${RUNTIME:-kata-runtime}"
+
+# Check if Sonobuoy is still running every 5 minutes.
+WAIT_TIME=300
+
+# Add a global timeout of 2 hours to stop the execution
+# in case Sonobuoy gets hanged.
+GLOBAL_TIMEOUT=$((WAIT_TIME*24))
+
+create_kata_webhook() {
+	pushd "${SCRIPT_PATH}/../../../kata-webhook"
+	# Create certificates for the kata webhook
+	./create-certs.sh
+
+	# Apply kata-webhook deployment
+	kubectl apply -f deploy/
+	popd
+}
+
+run_sonobuoy() {
+	# Get Sonobuoy repo
+	sonobuoy_repo="github.com/heptio/sonobuoy"
+	go get -u "$sonobuoy_repo"
```

Comment:
Not needed to pin the test?

Reply:
official documentation does not says that it is required to move to another commit/branch